be/src/common/daemon.cpp

```diff
@@ -352,6 +354,17 @@ void Daemon::block_spill_gc_thread() {
     }
 }
 
+void Daemon::je_purge_dirty_pages_thread() const {
+    do {
+        std::unique_lock<std::mutex> l(doris::MemInfo::je_purge_dirty_pages_lock);
+        doris::MemInfo::je_purge_dirty_pages_cv.wait(l);
```

Comment:
not wait inifinitely.
If the thread is stopped, should not wait any more.

Reply:
done